client/src/hooks/datafiles/mutations/useCompress.ts

```diff
@@ -129,20 +141,14 @@ function useCompress() {
               },
             });
           } else if (response.status === 'PENDING') {
-            dispatch({
-              type: 'DATA_FILES_SET_OPERATION_STATUS',
-              payload: { status: { type: 'SUCCESS' }, operation: 'compress' },
-            });
+            setStatus({ type: 'SUCCESS' });
             dispatch({
               type: 'ADD_TOAST',
               payload: {
                 message: 'Compress job submitted.',
               },
             });
-            dispatch({
-              type: 'DATA_FILES_SET_OPERATION_STATUS',
-              payload: { operation: 'compress', status: {} },
-            });
+            setStatus({});
```

Comment:
Is this `setStatus({})` call needed?

Reply:
I believe yes, it clears the status of the compress operation in state once the job has successfully submitted